superset-frontend/src/components/ObjectTags/ObjectTags.css

```
@@ -0,0 +1,149 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+.ant-tag {
+  color: black;
+}
+
+.react-tags {
+  position: relative;
+  display: inline-block;
+  padding: 1px 0 0 1px;
+  margin: 0 10px;
+  border: 0px solid #f5f5f5;
+  border-radius: 1px;
```

```suggestion
  border-radius: ${({ theme }) => theme.borderRadius}px;
```
Border-radius values should stay within the Superset theme, the current value is 4 as seen [here](https://github.com/apache/superset/blob/master/superset-frontend/packages/superset-ui-core/src/style/index.tsx#L55).